torch/_functorch/aot_autograd.py

```diff
@@ -3594,6 +3667,7 @@ def aot_dispatch_autograd(flat_fn, flat_args: List[Any], aot_config: AOTConfig,
                 + inner_meta.num_outputs
                 + inner_meta.num_intermediate_bases
                 + inner_meta.num_outputs_rng_offset
+                - inner_meta.num_mutated_graph_handled_indices
```

Comment:
instead of doing `inner_meta.num_mutated_inputs - inner_meta.num_mutated_graph_handled_indices`, can't we just do `inner_meta. num_mutated_runtime_inps`?